srcpkgs/LimeSuite/template

```diff
@@ -0,0 +1,85 @@
+# Template file for 'LimeSuite'
+pkgname=LimeSuite
+version=20.10.0
+revision=1
+build_style=cmake
+build_helper=cmake-wxWidgets-gtk3
+configure_args="
+ -DENABLE_SIMD_FLAGS=none
+ -DENABLE_UTILITIES=True
+ -DENABLE_LIME_UTIL=True
+ -DCMAKE_SKIP_RPATH=True
+ -DENABLE_DESKTOP=False
+ -DENABLE_SOAPY_LMS7=True
+ -DENABLE_PCIE_XILLYBUS=True
+ -DENABLE_QUICKTEST=True
+ -DENABLE_MCU_TESTBENCH=True
+ -DENABLE_FTDI=True
+ -DENABLE_FX3=True
+ -DENABLE_STREAM_UNITE=True
+ -DENABLE_EXAMPLES=True
+ -DENABLE_HEADERS=True
+ -DENABLE_GUI=True
+ -DENABLE_API_DOXYGEN=True
+ -DLIME_SUITE_EXTVER=${version}"
+hostmakedepends="doxygen"
+makedepends="gnuplot wxWidgets-gtk3-devel libusb-devel SoapySDR-devel libfreeglut-devel
+ fltk-devel glew-devel sqlite-devel libgomp-devel"
+depends="wget"
+short_desc="Driver and GUI for LimeSDR/LMS7002M-based SDR platforms"
+maintainer="classabbyamp <dev@kb6.ee>"
+license="Apache-2.0"
+homepage="https://myriadrf.org/projects/lime-suite/"
+distfiles="https://github.com/myriadrf/LimeSuite/archive/refs/tags/v${version}.tar.gz"
+checksum=f6d79dc67cb52a5aea839d1dc00e65f85367cb2c275d77f149833d32cf79b467
+shlib_provides="libLimeSuite.so.20.10-1"
```

Comment:
I don't think this is needed

Reply:
I believe it was added because of this: https://github.com/void-linux/void-packages/runs/4196787171?check_suite_focus=true#step:9:156